static/js/services/contact-summary.js

```diff
@@ -47,7 +47,7 @@ angular.module('inboxServices').service('ContactSummary',
       return summary;
     };
 
-    return function(contact, reports) { // jshint ignore:line
+    return function(contact, reports, lineage) { // jshint ignore:line
```

Comment:
Ugh yeah thanks! I completely forgot about this. I'll do it after this is merged.